mlx/backend/metal/kernels/bf16_math.h

```diff
@@ -369,7 +369,7 @@ instantiate_metal_math_funcs(
     return static_cast<otype>(__metal_simd_xor(static_cast<ctype>(data)));     \
   }
 
```

Comment:
Do we need to add 3.2 in `metal/device.cpp` as well?

Reply:
Yes definitely, good catch